README.md

```diff
@@ -14,20 +14,21 @@ On PC, you can play Pixel Wheels alone or with a friend.
 
 Great! For Linux, macOS and Windows you can get binaries from [Pixel Wheels page on itch.io][itch].
 
-_ARM-based Mac users: Note that Pixel Wheels does not run on ARM-based Macs for now. This will be fixed when [libgdx ARM-based Mac support][libgdx-arm-mac] is ready._
+_ARM-based Mac users: Note that Pixel Wheels (except for special build) does not run on ARM-based Macs for now. This will be added when [performance issues with libGDX][libgdx-arm-mac] will be resolved. You can download game with ARM support for testing at [this][version-to-test] link._
 
-[libgdx-arm-mac]: https://github.com/libgdx/libgdx/issues/6084
+[libgdx-arm-mac]: https://agateau.itch.io/pixelwheels/devlog/375260/mac-testers-wanted
+[version-to-test]: https://builds.agateau.com/pixelwheels/pixelwheels-0.22.0+update-libgdx-20220424T185600-ef5e324-mac.zip
 
 For Android you can install Pixel Wheels from [F-Droid][fd] or [Google Play][gplay].
 
-[itch]: http://agateau.itch.io/pixelwheels
-[fd]: https://f-droid.org/fr/packages/com.agateau.tinywheels.android/
-[gplay]: https://play.google.com/apps/testing/com.agateau.tinywheels.android
-
+[itch]: https://agateau.itch.io/pixelwheels
+[fd]: https://f-droid.org/packages/com.agateau.tinywheels.android/
+[gplay]: https://play.google.com/store/apps/details?id=com.agateau.tinywheels.android
+[builds-agateau]: https://builds.agateau.com/pixelwheels
```

Comment:
Can you move this line to the "master builds" section? I prefer when links are placed after they are used.

Reply:
Done.